crates/sui-indexer/src/indexer.rs

```diff
@@ -122,6 +122,14 @@ impl Indexer {
             spawn_monitored_task!(pruner.start(CancellationToken::new()));
         }
 
+        // If we already have chain identifier indexed (i.e. the first checkpoint has been indexed),
+        // then we persist protocol configs for protocol versions not yet in the db.
+        // Otherwise, we would do the persisting in `commit_checkpoint` while the first cp is
+        // being indexed.
+        if let Some(chain_id) = store.get_chain_identifier().await? {
+            store.persist_protocol_configs_and_feature_flags(chain_id)?;
+        }
+
```

Comment:
is this really necessary since it'll get picked up on the committer side?

Reply:
The target use case for this one is for when an up-to-date db (like our backfill dbs) wants to add protocol configs tables for the first time. If we don't do it here, we'll need to wait until epoch change.